pkg/controller/usersignup/usersignup_controller.go

```diff
@@ -51,15 +51,6 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 		return err
 	}
 
-	// Watch for changes to secondary resource Pods and requeue the owner UserSignup
-	err = c.Watch(&source.Kind{Type: &corev1.Pod{}}, &handler.EnqueueRequestForOwner{
```

Comment:
Why removing it completely?

I see it as:
1. Signup created. First reconcile. Check approved status or auto approval. Check if the corresponding MUR already exists. If not then set provisioning status, create MUR and exit.
If it does then status and exit.
2. When MUR is created it will trigger another reconcile. See 1.

Reply:
Now watching MasterUserRecord as the secondary resource.